### PR TITLE
CVE-2010-1157

### PR DESCRIPTION
**Pull request by zxw4320**

### CODE REVIEW

File: cves/CVE-2010-1157.yml [hunk 1]
```diff
@@ -44,36 +44,33 @@ description_instructions: |
...
+      the default behavior of the realm.
 vccs:
-  - commit: 2ee172f418591edd61cfd3dc9676753b281d1fe2
-    note: The git history for this directory only goes back to a branching of Tomcat 5.5.x, and does not reveal when original implementation was made
-  - commit:
-    note:
+  - commit: c4762411946378b1bdf758f16b15845f9646f9d5
+    note: |
+      The original VCC was part of modifying realm was recieved from the request
+      which resulted in the vulnerability.
```
Comment (rxv3774) [inline cves/CVE-2010-1157.yml]: I was not able to completely understand what you were trying to say here. Changing this sentence to something a bit more structured would make it easier to read, specifically, the "The original VCC was part of modifying realm was received" portion of it.

File: cves/CVE-2010-1157.yml [hunk 2]
```diff
@@ -44,36 +44,33 @@ description_instructions: |
...
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
-   - commit: 78e45a225c5c5564945191f7e421f8859885d63b
-     note: SVN rev 936541, from the Tomcat website.
-   - commit:
-     note:
+   - commit: a4de93b2aaf74fff7c1e194f7ce3d2228a9680b8
+     note: |
+      SVN rev 936539, from the Tomcat website. The fix was to add a variable for
+      the default behavior of the realm.
```
Comment (rxv3774) [inline cves/CVE-2010-1157.yml]: Using another word besides "realm" would help the reader understand what that means exactly. I worked on the same CVE so I understand the context, but others who haven't may not.

File: cves/CVE-2010-1157.yml [hunk 3]
```diff
@@ -105,12 +102,9 @@ unit_tested:
...
     Must be just "true" or "false".
-  answer: |
-    From looking at the git log, the code involved in the commits, and the bug reports,
-    I could not determine that any unit testing or automated testing was used on this
-    section of the code.
-  code: false
-  fix: false
+  answer: false
+  code:
+  fix:
```
Comment (rxv3774) [inline cves/CVE-2010-1157.yml]: I believe both the "fix" and "code" fields must be either true or false, as stated in the question above.

File: cves/CVE-2010-1157.yml [hunk 4]
```diff
@@ -91,7 +88,7 @@ upvotes_instructions: |
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 4
+upvotes:
```
Comment (rxv3774) [inline cves/CVE-2010-1157.yml]: Upvotes: 2

Comment (zxw4320) [reply cves/CVE-2010-1157.yml]: Updated

File: cves/CVE-2010-1157.yml (hunk 3 above)
Comment (zxw4320) [reply cves/CVE-2010-1157.yml]: You are correct. I have fixed this.

File: cves/CVE-2010-1157.yml (hunk 2 above)
Comment (zxw4320) [reply cves/CVE-2010-1157.yml]: I have updated this so that it understandable to a larger audience.

File: cves/CVE-2010-1157.yml (hunk 1 above)
Comment (zxw4320) [reply cves/CVE-2010-1157.yml]: Updated this sentence for clarity. Also reviewed the commit and determined that this was not the proper VCC.

**Pull request merged**